src/custom/state/orders/updaters/CancelledOrdersUpdater.ts

```diff
@@ -38,10 +38,15 @@ export function CancelledOrdersUpdater(): null {
   const fulfillOrdersBatch = useFulfillOrdersBatch()
 
   const updateOrders = useCallback(
-    async (chainId: ChainId) => {
-      // Filter orders created in the last 5 min, no further
+    async (chainId: ChainId, account: string) => {
+      const lowerCaseAccount = account.toLowerCase()
+      // Filter orders:
+      // - Owned by the current connected account
+      // - Created in the last 5 min, no further
       const pending = cancelledRef.current.filter(
-        (order) => Date.now() - new Date(order.creationTime).getTime() < CANCELLED_ORDERS_PENDING_TIME
+        ({ owner, creationTime }) =>
+          owner.toLowerCase() === lowerCaseAccount &&
+          Date.now() - new Date(creationTime).getTime() < CANCELLED_ORDERS_PENDING_TIME
```

Comment:
nit: cache `new Date(creationTime).getTime()` as a `const` for readability

Reply:
Good call, done  e9e22bbc5c3a8e405e22999420ae55f8818900ab